### PR TITLE
SWI-1695: Adding a missing environment variable

### PR DESCRIPTION
**Pull request by juliajanu**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -19,9 +19,10 @@ env:
   MANTECA_ACTIVE_NUMBER: ${{ secrets.MANTECA_ACTIVE_NUMBER }}
   MANTECA_APPLICATION_ID: ${{ secrets.MANTECA_APPLICATION_ID }}
   MANTECA_BASE_URL: ${{ secrets.MANTECA_BASE_URL }}
-  MANTECA_IDLE_NUMBER: ${{ secrets.MANTECA_IDLE_NUMBER }}
+  MANTECA_IDLE_NUMBER: ${{ secrets.MANTECA_IDLE_NUMBER }}
```
Comment (ajrice6713) [inline .github/workflows/deploy.yml:22]: trailing space 🤮

Comment (ckoegel) [reply .github/workflows/deploy.yml:22]: 🤮

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -37,9 +38,8 @@ jobs:
       
       - name: Check Release Tag Format
         run: |
-          re='^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(beta|alpha|rc))(\.[0-9]+)?$'
-          if ! [[ $RELEASE_VERSION =~ $re ]]; then
-            echo "Tag does not match expected regex pattern for beta releases ($re)"
+          if ! [[ $RELEASE_VERSION =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-(beta|alpha|rc))(\.[0-9][0-9]*)?$ ]]; then
+            echo 'Tag does not match expected regex pattern for beta releases (^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-(beta|alpha|rc))(\.[0-9][0-9]*)?$)'
```
Comment (ckoegel) [inline .github/workflows/deploy.yml:42]: what was wrong with the `$re`?

Comment (ajrice6713) [reply .github/workflows/deploy.yml:42]: Bash is what was wrong

Comment (juliajanu) [reply .github/workflows/deploy.yml:42]: bash would try to run the stuff within the parenthesis as a subshell command. This was just easier.

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -19,9 +19,10 @@ env:
   MANTECA_ACTIVE_NUMBER: ${{ secrets.MANTECA_ACTIVE_NUMBER }}
   MANTECA_APPLICATION_ID: ${{ secrets.MANTECA_APPLICATION_ID }}
   MANTECA_BASE_URL: ${{ secrets.MANTECA_BASE_URL }}
-  MANTECA_IDLE_NUMBER: ${{ secrets.MANTECA_IDLE_NUMBER }}
+  MANTECA_IDLE_NUMBER: ${{ secrets.MANTECA_IDLE_NUMBER }} 
   OPERATING_SYSTEM: ubuntu-latest
   CSHARP_VERSION: 7.0.x
+  DOTNET: 7.0.x
```
[the remark this answers is not in the document]
Comment (juliajanu) [reply .github/workflows/deploy.yml:25]: I guess not

**Review by ckoegel — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**